tests/testValueProviders.py

```diff
@@ -235,8 +243,13 @@ def checkOutput():
         for t in threads:
             t.join()
 
+
+        numpy.testing.assert_equal(opCompute.executionCount, 1)
+        assert opCache._dirty == False
```

Comment:
The reason for line 232 is that this test wants to force the cache to request the data from the upstream source, not to serve it out of memory.  I think it makes sense, and I think the test is fixed now.